packages/docs-ui/src/private/touchable/virtualTouchableRules.ts

```diff
@@ -0,0 +1,9 @@
+import { hitArea } from './hitArea';
+
+export const virtualTouchableRules = {
```

Comment:
```suggestion
import type { CSSProperties } from '@vanilla-extract/css';

import { hitArea } from './hitArea';

export const virtualTouchableRules: CSSProperties = {
```

Without it, having `transformers: 'rollout'` is not an error. With the change:

<img width="553" alt="error with transformers" src="https://github.com/seek-oss/braid-design-system/assets/3297808/efd54344-2ad6-4701-97ee-d17b5d4c1670">